components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -257,15 +257,24 @@ impl Recorder {
             }
             let sampled = sample.contained + sample.left + sample.right;
             if (sample.left + sample.right) == 0 || sampled < sample_threshold {
+                LOAD_BASE_SPLIT_EVENT
+                    .with_label_values(&["no_enough_key"])
+                    .inc();
                 continue;
             }
             let diff = (sample.left - sample.right) as f64;
             let balance_score = diff.abs() / (sample.left + sample.right) as f64;
             if balance_score >= split_balance_score {
+                LOAD_BASE_SPLIT_EVENT
+                    .with_label_values(&["no_balance_key"])
```

Comment:
can we add a histogram for the `balance_score` and  `contained_score`?

Reply:
good idea